File: tutorials/testing/python/shakespeare_snippet.py
from bridges.bridges import *
from bridges.data_src_dependent.data_source import *
import sys
import random

def main():
    # create the Bridges object, set credentials
    bridges = Bridges(YOUR_ASSSIGNMENT_NUMBER, "YOUR_USER_ID", "YOUR_API_KEY")


    # get the shakespeare data
    my_list = get_shakespeare_data()

    # pick a work at random
    work1 = my_list[random.randrange(len(my_list))]

    # print to console
    print(work1.title)
    print(work1.type)
    print(work1.text[0:min(100, len(work1.text))])


if __name__ == "__main__":
    main()

File: tutorials/testing/java/shakespeare_snippet.java
import java.util.List;
import java.util.Random;
import bridges.connect.Bridges;
import bridges.connect.DataSource;
import bridges.data_src_dependent.Shakespeare;

// a program fragment to access the Shakespeare data (plays, poems)
public class shakespeare_snippet {
	public static void main(String[] args) throws Exception {

		//create the Bridges object
		Bridges bridges = new Bridges(YOUR_ASSIGNMENT_NUMBER, "YOUR_USER_ID", "YOUR_API_KEY");

		// Get a List of Shakespeare objects from Bridges
		DataSource ds = bridges.getDataSource();
		List<Shakespeare> mylist = ds.getShakespeareData();

		// Inspect a random Shakespeare object
		Shakespeare work1 = mylist.get((new Random()).nextInt(mylist.size()));
		System.out.println(work1.getTitle());
		System.out.println(work1.getType());
		System.out.println(work1.getText().substring(0, Math.min(100, work1.getText().length())) + "...");
	}
}
